### DISCUSSION TITLE
No "terrain_dir" in minitaur_randomize_terrain_gym_env

### DISCUSSION THREAD

**Discussion by abhiksingla** [bulletphysics/bullet3#3763] [General]
Hi,
The environment [minitaur_randomize_terrain_gym_env] should randomly choose from different terrain obj files and load at reset. But it seems that the terrain obj files are not included in the repository. Is there a workaround to this problem ?

**Comment by floodsung** [comment]
The same issue!
Looking forward to having terrain_obj in the git!
Thank you
